packages/create-cloudflare/src/workers.ts

```diff
@@ -147,12 +151,14 @@ async function updateFiles(ctx: Context) {
 	};
 
 	// update files
-	contents.packagejson.name = ctx.project.name;
+	if (contents.packagejson.name === "<TBD>") {
+		contents.packagejson.name = ctx.project.name;
+	}
 	contents.wranglertoml = contents.wranglertoml
```

Comment:
Would a `toml` parser not make more sense here to avoid any quirkiness that comes with regex replacing?

Reply:
A toml parser would be more effective for bizarre templates, but they are not very useful for manlpulating toml, since you lose comments and layout. In these templates, the toml should not be too messy.

Reply:
Comments make a lot of sense! Thanks for the insight.